4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4896 @@
+{
```

Comment:
<p>Not sure it's useful to generate the data a third time, I think they got it and that will save time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>